### PR TITLE
185 Show completed nodes

### LINKED ISSUE #185: Show completed nodes - videos in backpack

### PR DESCRIPTION
**Pull request by anhduc130**
Fixes #185

### CODE REVIEW

**Comment by anhduc130** [comment]
I believe this branch was branched off from `149-tyde-menu`. However, this branch was later synced up with `tyde` in order to use Vuex and all the updated changes. That's why I put the base branch as `tyde`.

**Comment by wynnset** [comment]
This PR should technically be merged into the tyde-menu branch, not master -unless we merge that in to master first

File: templates/vue/src/components/tyde/TydeLog.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<template>
+  <li class="log">
+    <img :src="log.imageURL" />
+    <div class="log-details">
+      <div>{{log.title}}</div>
+      <div>{{log.description}}</div>
```
Comment (narendrasss) [inline templates/vue/src/components/tyde/TydeLog.vue]: Can you change this to use semantic tags? i.e. `h1` for titles and `p` for text/description. It makes it more obvious what's supposed to go there and it's better for accessibility in the future.

Comment (anhduc130) [reply templates/vue/src/components/tyde/TydeLog.vue]: Fixed this. Thanks.

**Comment by wynnset** [comment]
@anhduc130 could you please fix the merge conflicts here now that the tyde-menu branch is merged?

**Pull request merged**